docs/ism/index-rollups.md

```diff
@@ -84,3 +84,407 @@ GET target_index/_search
+              "sku": "ZO0085200852",
+              "taxless_price": 10.99,
+              "unit_discount_amount": 0,
+              "min_price": 5.06,
+              "_id": "sold_product_581553_17221",
+              "discount_amount": 0,
+              "created_on": "2016-12-24T03:56:10+00:00",
+              "product_name": "Snood - rose",
+              "price": 10.99,
+              "taxful_price": 10.99,
+              "base_unit_price": 10.99
+            }
+          ],
+          "sku": [
+            "ZO0064500645",
+            "ZO0085200852"
+          ],
+          "taxful_total_price": 35.98,
+          "taxless_total_price": 35.98,
+          "total_quantity": 2,
+          "total_unique_products": 2,
+          "type": "order",
+          "user": "selena",
+          "geoip": {
+            "country_iso_code": "MA",
+            "location": {
+              "lon": -8,
+              "lat": 31.6
+            },
+            "region_name": "Marrakech-Tensift-Al Haouz",
+            "continent_name": "Africa",
+            "city_name": "Marrakesh"
+          },
+          "event": {
+            "dataset": "sample_ecommerce"
+          }
+        }
+      }
+    ]
+  }
+}
+...
+```
+
+Create an index rollup job:
+
+```json
+PUT _opendistro/_rollup/jobs/example
+{
+  "rollup": {
+    "enabled": true,
+    "schedule": {
+      "interval": {
+        "period": 1,
+        "unit": "Minutes",
+        "start_time": 1602100553
+      }
+    },
+    "last_updated_time": 1602100553,
+    "description": "An example policy that rolls up the sample ecommerce data",
+    "source_index": "kibana_sample_data_ecommerce",
+    "target_index": "example_rollup",
+    "page_size": 1000,
+    "delay": 0,
+    "continuous": false,
+    "dimensions": [
+      {
+        "date_histogram": {
+          "source_field": "order_date",
+          "fixed_interval": "60m",
+          "timezone": "America/Los_Angeles"
+        }
+      },
+      {
+        "terms": {
+          "source_field": "customer_gender"
+        }
+      },
+      {
+        "terms": {
+          "source_field": "geoip.city_name"
+        }
+      },
+      {
+        "terms": {
+          "source_field": "geoip.region_name"
+        }
+      },
+      {
+        "terms": {
+          "source_field": "day_of_week"
+        }
+      }
+    ],
+    "metrics": [
+      {
+        "source_field": "taxless_total_price",
+        "metrics": [
+          {
+            "avg": {}
+          },
+          {
+            "sum": {}
+          },
+          {
+            "max": {}
+          },
+          {
+            "min": {}
+          },
+          {
+            "value_count": {}
+          }
+        ]
+      },
+      {
+        "source_field": "total_quantity",
+        "metrics": [
+          {
+            "avg": {}
+          },
+          {
+            "max": {}
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+Query the target index:
```

Comment:
What should the reader be noting about the response here? I think we need 1-2 sentences of explanation for every one of these steps.